test/extended/framework/client.go

```diff
@@ -0,0 +1,358 @@
+package framework
```

Comment:
Add here where the client came from (this is a smaller set from the original openshift/origin repo, without the Ginkgo specifics)